.github/workflows/continous-delivery.yml

```
@@ -136,4 +136,8 @@ jobs:
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
           script: |
-            await github.request(`DELETE /repos/${{ github.repository }}/git/refs/tags/v${{ github.run_number }}-rc`);
+            const release = await github.request(`GET /repos/${{ github.repository }}/releases/tags/v${{ github.run_number }}-rc`);
+            if (release) {
+              await github.request(`DELETE /repos/${{ github.repository }}/releases/${release.id}`);
+              await github.request(`DELETE /repos/${{ github.repository }}/git/refs/tags/v${{ github.run_number }}-rc`);
```

si je reprends comme la ligne 94 du même fichier on peut même écrire :

```suggestion
                     await github.request(`DELETE /repos/${{ github.repository }}/git/refs/tags/${release.tag_name}`);
```